apps/zbugs/shared/schema.ts

```diff
@@ -12,12 +12,30 @@ import {
 } from '@rocicorp/zero';
 import type {Role} from './auth.ts';
 
+const created = number().default({
+  insert: {
```

Comment:
did you guys discuss a shorthand for both client & server? e.g., `insert: () => Date.now()`

Reply:
We didn't - @aboodman what do you think about this?

I like this simplified API.